### PR TITLE
Partner Script Disassembly

### PR DESCRIPTION
**Pull request by JoshDuMan**

### CODE REVIEW

**Comment by BowserSlug** [comment]
✅ This PR fixes 3 warnings!

File: src/world/partner/kooper.c [hunk 1]
```diff
@@ -71,6 +71,14 @@ ApiStatus func_802BD228_31B248(Evt* script, s32 isInitialCall) {
     return partner_get_out(kooper) ? ApiStatus_DONE1 : ApiStatus_BLOCK;
 }
 
+EvtSource world_kooper_take_out = {
+    EVT_CALL(func_802BD228_31B248)
+    EVT_RETURN
+    EVT_END
+};
+
+unkPartnerStruct* D_802BEB60_31CB80 = 0x802BEC80;
```
Comment (nanaian) [inline src/world/partner/kooper.c:80]: `// TODO rawptr` this

File: src/world/partner/parakarry.c [hunk 2]
```diff
@@ -32,6 +31,14 @@ ApiStatus func_802BD148_3196B8(Evt* script, s32 isInitialCall) {
     return partner_get_out(parakarry) ? ApiStatus_DONE1 : ApiStatus_BLOCK;
 }
 
+EvtSource world_parakarry_take_out = {
+    EVT_CALL(func_802BD148_3196B8)
+    EVT_RETURN
+    EVT_END
+};
+
+struct unkPartnerStruct* D_802BEAAC_31B01C = 0x802BEBC8;
```
Comment (nanaian) [inline src/world/partner/parakarry.c:40]: ", also probably don't need the `struct` keyword

File: src/world/partner/watt.c [hunk 3]
```diff
@@ -79,6 +81,16 @@ ApiStatus func_802BD27C_31CDEC(Evt* script, s32 isInitialCall) {
     }
 }
 
+EvtSource world_watt_take_out = {
+    EVT_CALL(func_802BD27C_31CDEC)
+    EVT_RETURN
+    EVT_END
+};
+
+unkPartnerStruct* D_802BE274_31DDE4 = 0x802BE318;
```
Comment (nanaian) [inline src/world/partner/watt.c:90]: "

**Review by nanaian — CHANGES_REQUESTED** [review]
Awesome work

File: src/world/partner/kooper.c (hunk 1 above)
Comment (ethteck) [inline src/world/partner/kooper.c:80]: this symbol may be bss. you can find out by seeing if it goes past the end of the "natural" overlay length (get the rom size of the overlay and add the ram start to the size to get the ram end. if the symbol is past that, it's either bss or from a different overlay)

Comment (ethteck) [reply src/world/partner/kooper.c:80]: I see there's 1 or 2 more of these. the solution would be to add them to undefined_syms and then extern a new symbol for them

**Review by ethteck — CHANGES_REQUESTED** [review]

**Review by Brotenko — APPROVED** [review]
Very nice PR! LGTM, besides the ongoing BSS discussion

**Pull request merged**